kratos/sources/variables.cpp

```diff
@@ -973,6 +977,8 @@ void KratosApplication::RegisterVariables() {
     Serializer::Register("Node3D", NodeType());
     Serializer::Register("DofDouble", Dof<double>());
 
+    Serializer::Register("MasterSlaveConstraint", MasterSlaveConstraint());
```

Comment:
I think we shall also register it in the KratosComponents. Isn't this handled by the macro you added in the define?

Reply:
Yes I think so, it is done by the macro.